keymap.cson

```diff
@@ -23,3 +23,14 @@
 # This file uses CoffeeScript Object Notation (CSON).
 # If you are unfamiliar with CSON, you can read more about it here:
 # https://github.com/bevry/cson#what-is-cson
+
+# window/pane navigation
+'body, atom-workspace atom-text-editor:not([mini])':
+  'cmd-h': 'window:focus-previous-pane'
+  'cmd-l': 'window:focus-next-pane'
+  'cmd-shift-v': 'pane:split-right'
+  'cmd-shift-s': 'pane:split-down'
```

Comment:
I use save-as very rarely, I'm happy using cmd+? for that (cmd+?, "save", ctrl+n, ctrl+n,RET) as I use it so rarely.